helm-chart/templates/_helpers.tpl

```diff
@@ -0,0 +1,68 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "kubeshark.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "kubeshark.fullname" -}}
+{{- if .Values.fullnameOverride }}
```

Comment:
`fullnameOverride` field does not exist in `values.yaml`.

Reply:
The `fullnameOverride` and `nameOverride` fields in Helm charts provide a mechanism to customize and control the naming of Kubernetes resources generated by the chart during deployment. These fields are particularly useful when you want to fine-tune the naming of resources to align with your specific project or organization's naming conventions, or to prevent naming conflicts in multi-environment deployments.
This can be especially valuable in scenarios where you are deploying multiple instances of the same chart in the same cluster or across different clusters or slight variations based on the deployment context.

We don't have it now on `values.yaml`, but the inclusion of these terms is willingly embraced and should be included in the future.